projects/plugins/jetpack/extensions/blocks/subscriber-login/subscriber-login.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * Subscriber Login Block.
+ *
+ * @since 0.0.1
+ *
+ * @package automattic/jetpack
+ */
+
+namespace Automattic\Jetpack\Extensions\Subscriber_Login;
+
+use Automattic\Jetpack\Blocks;
+use Automattic\Jetpack\Extensions\Premium_Content\Subscription_Service\Abstract_Token_Subscription_Service;
+use Automattic\Jetpack\Status\Host;
+use Jetpack_Gutenberg;
+use Jetpack_Memberships;
+use Jetpack_Options;
+
+/**
+ * Registers the block for use in Gutenberg
+ * This is done via an action so that we can disable
+ * registration if we need to.
+ */
+function register_block() {
+	if ( ! \Jetpack::is_module_active( 'subscriptions' ) || ! class_exists( '\Jetpack_Memberships' ) ) {
+		return;
+	}
```

Comment:
Just for reference in case there are other checks in the subscriptions block that make sense to add here too:

https://github.com/Automattic/jetpack/blob/20e0d8fe09b53c71910a9e31e68b28d7618ac6b0/projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php#L38-L80